arrows/stanag/stanag_util.cxx

```diff
@@ -25,6 +25,19 @@ trim_whitespace( std::string input )
   return input.substr( str_begin, ( str_end - str_begin ) + 1 );
 }
 
+// ----------------------------------------------------------------------------
+float
+float_to_binary_angle( float value, int n )
+{
+  return value * 1.40625 * ( 1 / pow( 2, n - 8 ) );
```

Comment:
One thing you might like to consider here (and for the SA case) is that `n` will only ever be one of two values (16 or 32). So you can turn this into a straight scaling factor, either with `if (n == ...)` construct or by defining two functions for each case (e.g. `float_to_binary_angle_16()`). Then you don't have the pay the cost of the `pow()` call each time.